src/components/Colors/Color/Color.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ComponentPropsWithoutRef } from "react";
+
+export interface ColorRectangleProps extends ComponentPropsWithoutRef<"div"> {
+  color: string;
+}
+
+export function Color(props: ColorRectangleProps) {
+  return (
+    <div className="flex ">
+      <div className={`w-[156px] mb-2 h-[40px] ${props.color}`}></div>
```

Comment:
Join **h** & **w** helps to better read. We have some preset number that represents your choice:

```suggestion
      <div className={`w-40  h-10 mb-2 ${props.color}`}></div>
```